modules/upi-removing-hosts-from-a-upi-cluster.adoc

```diff
@@ -74,6 +74,13 @@ openshift-machine-api   controller1   externally provisioned              true
 openshift-machine-api   worker1       deprovisioning                      true             57m
 ----
 
+. Delete the host by running the following command:
```

Comment:
Deleting a host triggers deprovisioning so you no longer need the previous action (editing BMH). It may be worth adding a note that the host deletion takes time because it waits for deprovisioning.

Reply:
@dtantsur while testing found that editing starts the deprovisoning of the bmh and when it's done the bmh is in `available` state, hence the delete step of the bmh. Step 3 should be to delete the host when state changes to `available` (happens immediatley). As you suggested a note could be added that deleting can also be done without editing, but the command will take time to complete as it waits for deprovisioning.

Reply:
Updated as suggested.